tests/test_ptcDataset.py

```diff
@@ -98,11 +98,11 @@ def test_ptcDatset(self):
                     localDataset.ptcTurnoff[ampName] = localDataset.rawMeans[ampName][-1]
 
                     localDataset.covariances[ampName] = np.full(
-                        (nSignalPoints, nSideCovMatrix, nSideCovMatrix), np.nan).tolist()
```

Comment:
I see these match the values in the FULLCOVARIANCE tests.  Are these just the new correct values for the test?  I think I just don't see why these are switching from NaN to a value (probably just me not spending enough time thinking through the test).

Reply:
Before, the code only calculated/measured the covariances when fitting the `FULLCOVARIANCE` model. Then, when we decided to split the PTC pipeline into `extract` and `solve`, we decided that we would measure the covariances during `extract`, regardless of the model being fit in `solve`. So now these two models ( `EXPAPROXIMATION` and `POLYNOMIAL`  ) are expected to have a measured covariances attribute.